qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -33,57 +33,95 @@
-    drive_format_stg1 = scsi-hd
-    remove_image_stg0 = yes
-    remove_image_stg1 = yes
-    force_create_image_stg0 = yes
-    force_create_image_stg1 = yes
-    serials += " vs"
-    serial_type_vs = virtserialport
-    balloon = balloon0
-    balloon_dev_devid = balloon0
-    balloon_dev_add_bus = yes
-    inputs = input1
-    input_dev_bus_type_input1 = virtio
-    input_dev_type_input1 = mouse
-    filesystems = fs
-    fs_driver = virtio-fs
-    fs_source_type = mount
-    fs_source_dir = virtio_fs_test/
-    force_create_fs_source = yes
-    remove_fs_source = yes
-    fs_target = 'myfs'
-    fs_driver_props = {"queue-size": 1024}
-    mem = 4096
-    mem_devs = mem1
-    backend_mem_mem1 = memory-backend-file
-    mem-path_mem1 = /dev/shm
-    size_mem1 = 4G
-    use_mem_mem1 = no
-    share_mem = yes
-    guest_numa_nodes = shm0
-    numa_memdev_shm0 = mem-mem1
-    numa_nodeid_shm0 = 0
-    fs_binary_extra_options = " -o cache=auto"
     Win10, Win2016, Win2019:
         check_qemufwcfg = yes
     Win8, Win2012, Win2012..r2:
         need_reboot = yes
+    nic_model_nic1 = rtl8139
+    q35:
+        nic_model_nic1 = e1000e
+    variants:
+        - @all_drivers:
+            nics += " nic2"
+            nic_model_nic2 = virtio
+            no_virtio_rng:
+                virtio_rngs += " rng0"
+                backend_rng0 = rng-builtin
+                backend_type = builtin
+            images += " stg0 stg1"
+            image_name_stg0 = "images/stg0"
+            image_name_stg1 = "images/stg1"
+            image_size_stg0 = 5G
+            image_size_stg1 = 10G
+            drive_format_stg0 = virtio
+            drive_format_stg1 = scsi-hd
+            remove_image_stg0 = yes
+            remove_image_stg1 = yes
+            force_create_image_stg0 = yes
+            force_create_image_stg1 = yes
+            serials += " vs"
+            serial_type_vs = virtserialport
+            balloon = balloon0
+            balloon_dev_devid = balloon0
+            balloon_dev_add_bus = yes
+            inputs = input1
+            input_dev_bus_type_input1 = virtio
+            input_dev_type_input1 = mouse
+            filesystems = fs
+            fs_driver = virtio-fs
+            fs_source_type = mount
+            fs_source_dir = virtio_fs_test/
+            force_create_fs_source = yes
+            remove_fs_source = yes
+            fs_target = 'myfs'
+            fs_driver_props = {"queue-size": 1024}
+            mem = 4096
+            mem_devs = mem1
+            backend_mem_mem1 = memory-backend-file
+            mem-path_mem1 = /dev/shm
+            size_mem1 = 4G
+            use_mem_mem1 = no
+            share_mem = yes
+            guest_numa_nodes = shm0
+            numa_memdev_shm0 = mem-mem1
+            numa_nodeid_shm0 = 0
+            fs_binary_extra_options = " -o cache=auto"
+            driver_test_names = 'rng iozone'
+            driver_test_params = '{};{'images': 'stg0 stg1'}'
```

Comment:
I would recommend `driver_test_params_rng = '{}'`, `driver_test_params_iozone = {'images': 'stg0 stg1'}`.
And in the script you might have:
```
driver_test_params = params.get('driver_test_params_%s' % test_name, '{}')
```
Thus even `driver_test_params_rng = '{}'` could be dropped.

Reply:
@vivianQizhu  Thanks，that's a good idea. updated it, thanks